packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -130,6 +130,15 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     <>
       <div ref={messageRef}>
         <Chat.Message
+          aria-label={_formatString(strings.messageContentAriaLabel, {
```

Comment:
can you build this string outside of this large body of nested html tags and memoize with useMemo also?

Reply:
updated